compiler/src/dmd/backend/machobj.d

```diff
@@ -2690,9 +2690,9 @@ int elf_align(targ_size_t size, int foffset)
 @trusted
 void MachObj_moduleinfo(Symbol *scc)
 {
-    int align_ = I64 ? 3 : 2; // align to _tysize[TYnptr]
+    int p2align = I64 ? 3 : 2; // align to _tysize[TYnptr]
```

Comment:
3 is not a power of 2...

Reply:
It is a power of 2 byte alignment.  (i.e: `.p2align 3` is 8 bytes).